content/docs/1.7.0/v2-data-engine/features/node-disk-support.md

```diff
@@ -0,0 +1,156 @@
+
+1. Add the NVMe disk to `spec.disks` of `node.longhorn.io`
+   ```
+   nvme-disk:
+     allowScheduling: true
+     diskType: block
+     diskDriver: auto
+     evictionRequested: false
+     path: 0000:05:00.0
+     storageReserved: 0
+     tags: []
+   ```
+
+1. Check the `status.diskStatus`. The disk should be detected without errors, and the diskDriver should be set to `nvme`.
+
+> Note: **Alternative Disk Configuration**
+> 
+> If you add the disk using a different path, such as:
+>  ```
+>  nvme-disk:
+>    allowScheduling: true
+>    diskType: block
+>    diskDriver: auto
+>    evictionRequested: false
+>    path: /dev/nvme1n1
+>    storageReserved: 0
+>    tags: []
+>  ```
+> In this case, the disk will be managed by the aio bdev driver, and the `node.disks[i].diskDriver` will be set to `aio`.
+
+### Using VirtIO Disks
+
+The steps are similar to NVMe disks.
+
+1. List the disks
+   First, identify the VirtIO disks available on your system by running the following command:
+   ```
+   # ls -al /sys/block/
+   ```
+
+   Example output:
+   ```
+   lrwxrwxrwx  1 root root 0  Jul  30 12:20 loop0 -> ../devices/virtual/block/loop0
+   lrwxrwxrwx  1 root root 0 Feb 22 14:04 vda -> ../devices/pci0000:00/0000:00:02.3/0000:04:00.0/virtio2/block/vda
+   lrwxrwxrwx  1 root root 0 Feb 22 14:24 vdb -> ../devices/pci0000:00/0000:00:02.6/0000:07:00.0/virtio5/block/vdb
+   ```
+
+1. Get the BDF of the VirtIO disk
+   Identify the BDF of the VirtIO disk /dev/vdb. From the example above, the BDF is 0000:07:00.0.
+
+1. Add the NVMe disk to `spec.disks` of `node.longhorn.io`
+   ```
+   nvme-disk:
+     allowScheduling: true
+     diskType: block
+     diskDriver: auto
+     evictionRequested: false
+     path: 0000:07:00.0
+     storageReserved: 0
+     tags: []
+   ```
+
+1. Check the `status.diskStatus`. The disk should be detected without errors, and the diskDriver should be set to `virtio-blk`.
+
+> Note: **Alternative Disk Configuration**
+> 
+> If you add the disk using a different path, such as:
+>  ```
+>  nvme-disk:
+>    allowScheduling: true
+>    diskType: block
+>    diskDriver: auto
+>    evictionRequested: false
+>    path: /dev/vdb
+>    storageReserved: 0
+>    tags: []
+>  ```
+> In this case, the disk will be managed by the aio bdev driver, and the `node.disks[i].diskDriver` will be set to `aio`.
+
+
+### Using AIO Disks
+
+When neither NVMe nor VirtIO drivers can manage a disk, Longhorn will default to using the aio bdev driver. Users can also manually configure this.
+
+1. Add the disk to `spec.disks` of `node.longhorn.io`
+    ```
+    default-disk-loop:
+      allowScheduling: true
+      diskDriver: aio
+      diskType: block
+      evictionRequested: false
+      path: /dev/loop12
+      storageReserved: 0
+      tags: []
+    ```
+
+2. Check node.status.diskStatus. The disk should be detected without errors, and the `node.disks[i].diskDriver` will be set to aio.
+
+
```

Comment:
I should add a section where we say that, using nvme or virtio diskDriver, the disk will be detached by kernel driver and so the block device `/dev/nvmeXnY` or `/dev/vdbX` will disappear. WDYT?

Reply:
Sounds good. I will add it.

Reply:
Added. Thank you.